src/send/SendConfirmation.test.tsx

```diff
@@ -183,7 +199,26 @@ describe('SendConfirmation', () => {
     expect(getElementText(totalComponent)).toEqual('₱1.36')
   })
 
-  it('renders correctly for send payment confirmation with cEUR fees', async () => {
+  it('renders correctly for send payment confirmation, sending cUSD, fee in CELO (new UI)', async () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
+    const { getByText, getByTestId } = renderScreen()
+
+    fireEvent.press(getByText('feeEstimate'))
+
+    await act(() => {
+      jest.runAllTimers()
+    })
+
+    const feeComponent = getByTestId('feeDrawer/SendConfirmation/totalFee')
+    expect(getElementText(feeComponent)).toEqual('0.004 CELO')
+
+    const totalComponent = getByTestId('TotalLineItem/Total')
+    expect(getElementText(totalComponent)).toEqual('~1.02 cUSD')
+  })
+
+  // TODO: Need to make for old and new
+  it('renders correctly for send payment confirmation with cEUR fees (old UI)', async () => {
+    jest.mocked(getFeatureGate).mockReturnValue(false)
```

Comment:
nit: you can set a default mock for `getFeatureGate` in the `jest.mock('src/statsig')` line at the top of this file so you don't have to manually mock the return value in quite as many tests.